command/members/members_test.go

```diff
@@ -206,8 +205,6 @@ func zip(t *testing.T, k, v []string) map[string]string {
 }
 
 func TestSortByMemberNamePartitionAndSegment(t *testing.T) {
-	lib.SeedMathRand()
```

Comment:
This is no longer needed anywhere because the math/rand package will automatically generate a random seed starting in 1.20.